lib/matplotlib/backends/backend_macosx.py

```diff
@@ -160,6 +160,7 @@ def prepare_configure_subplots(self):
         toolfig = Figure(figsize=(6, 3))
         canvas = FigureCanvasMac(toolfig)
         toolfig.subplots_adjust(top=0.9)
+        # Need to keep a reference to the tool.
         tool = SubplotTool(self.canvas.figure, toolfig)
```

Comment:
```suggestion
        _tool = SubplotTool(self.canvas.figure, toolfig)
```
The underscore prefix is commonly used as "intentionally unused"; e.g. Pycharm does not mark the variable as unused anymore with this change.

Reply:
sure, done